README.md

```diff
@@ -356,6 +356,17 @@ config.coverage :CI do |coverage|
 end
 ```
 
+### Ignoring Files
+
+Teaspoon allows defining files and directories to ignore when performing coverage instrumentation.
+
+The below example will ensure that the files within `/test/javascripts` and `/vendor/assets` are not intrumented by Istanbul and will not show up in the coverage results.
+
+```ruby
+config.coverage_ignored_files = ["/test/javascripts", "/vendor/assets"]
```

Comment:
I was just thinking about the matcher and I think it makes more sense here to use a similar style to the `no_coverage` option. Then you would be able to ignore specific directories or files. I don't think with the matcher it could ignore both `spec/javascripts` and `vendor/assets`?